### PR TITLE
Play icon function

### PR DESCRIPTION
**Pull request by marezloum**
Done!

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -1081,54 +1081,60 @@ footer {
...
-  &:hover {
-    &::before {
-      visibility: visible;
-      opacity: 1;
-    }
-    &::after {
-      visibility: hidden;
-      transform: rotate(-360deg);
-      opacity: 0.0001;
+    .playControl {
```
Comment (alisalehi6990) [inline css/style.scss:1089]: Better to have style to make the cursor as a pointer:
 cursor: pointer;

**Review by alisalehi6990 — CHANGES_REQUESTED** [review]

File: css/style.scss (hunk 1 above)
Comment (marezloum) [reply css/style.scss:1089]: Done.

File: single-video.html [hunk 2]
```diff
@@ -400,7 +400,11 @@ <h2>Blog post title goes here</h2>
                 >
                   
               </video>
+              <div class="playPauseIcon">
+                <a class="playControl"></a>
               </div>
+              </div>
```
Comment (alisalehi6990) [inline single-video.html]: Extra close tag. Need to be removed
```suggestion
              
```

**Review by alisalehi6990 — CHANGES_REQUESTED** [review]

File: single-video.html (hunk 2 above)
Comment (marezloum) [reply single-video.html]: CR done.

**Pull request merged**